apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -43,8 +43,7 @@ import {
   DEVICE_STATUS_POLLING_INTERVAL_MS,
   DEVICE_STATUS_POLLING_TIMEOUT_MS,
   MAX_BALLOT_BOX_CAPACITY,
-  RESET_AFTER_JAM_DELAY_MS,
-  SUCCESS_NOTIFICATION_DURATION_MS,
+  NOTIFICATION_DURATION_MS,
```

Comment:
This file's diff is mostly whitespace; recommend turning whitespace diff off to view